webapp/components/test/wpt-metadata.html

```diff
@@ -44,9 +46,68 @@
       assert.equal(appFixture.displayedMetadata[0].url, 'https://bug1');
       assert.equal(appFixture.displayedMetadata[0].product, 'chrome');
     });
+    test('simple case with different pending metadata tests', () => {
+      appFixture.searchResults = [{ test: '/foo.html' }];
+      appFixture.pendingMetadata = {
+        '/bar.html': [
+          {
+            url: 'bug1',
+            product: 'chrome'
+          }
+        ]
+      };
+      appFixture.mergedMetadata = {
+        '/foo.html': [
+          {
+            url: 'bug1',
```

Comment:
Should a different url and product be used here, to rule out the test passing if pending and merged are mixed up?